docs/install-olm.md

```diff
@@ -0,0 +1,64 @@
+# How do I install OLM?
+
+The OLM is installable on Kubernetes clusters. It is already installed and available by default on 4.x OpenShift 
+clusters. For the following instructions to work, you must have a Kubernetes cluster running and the `kubectl` 
+command-line tool is able to find the information it needs to communicate with the API server of that cluster. For more 
+information about configuring `kubectl`, please visit 
+[here](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/).
+
+### Install locally with minikube
+
+To install OLM locally on a minikube cluster, you need to spawn a 1.16.0 minikube cluster and then follow the 
+instructions installing OLM from releases or GitHub repositories:
+
+```bash
+minikube start  --kubernetes-version=1.16.0 
```

Comment:
Earlier you had mentioned that this document requires a running kubernetes cluster. In this step you instruct users how to start minikube. I do not believe we need to tell users how to start minikube.